lua/entities/gmod_wire_expression2/core/custom/constraintcore.lua

```diff
@@ -92,6 +92,8 @@ local countLookup = {
 
 local function checkEnts(self, ent1, ent2)
 
+	if not (ent1 and ent2) then return self:throw("Invalid entity (nil)!", false) end
```

Comment:
The (nil) isn't necessary, E2 users won't know what that is so it'll just confuse them.

I think `not ent1 or not ent2` would also be more clear logic